app/containers/Home/saga.js

```
@@ -0,0 +1,40 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { getTrendingTracks } from '@services/musicApi';
+import { homeActionCreators, homeActionTypes } from './reducer';
+
+function extractTracksFromResponse(response) {
+  if (!response) return null;
```

ESLint

[error] Expected { after 'if' condition.

(curly)